src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAloneOrSingleline.java

```diff
@@ -209,4 +209,25 @@ public void codeAfterLastRightCurly() {
         }; // violation
         for (int i = 0; i < 1; i++) { new Object(); }; // violation
     }
+
+    public void testMethod() {}; //violation
+
+    public void testMethod1() {
+    }; //violation
+
+    public class TestClass {}; //violation
```

Comment:
Why is this a violation? The method is singleline, shouldn't that take precedence over alone?

Reply:
@rnveach Not a violation. I have fixed it.

Reply:
@sd1998 The same thing is at line 224 for the nested class.
> `public TestClass2() {}; //violation`

Is this isn't a violation, please look over the inputs and make sure all violations say violation and ones that aren't a violation don't say it.

Reply:
Done